pkgs/top-level/python-packages.nix

```diff
@@ -2406,7 +2406,7 @@ in {
 
   pkginfo = callPackage ../development/python-modules/pkginfo { };
 
-  pre-commit = callPackage ../development/python-modules/pre-commit { };
+  pre-commit = disabledIf isPy27 (callPackage ../development/python-modules/pre-commit { });
```

Comment:
Not sure if this right thing to do, the top comment in this file states that python cli utils don't belong to that file, so I think we should remove this line.

Reply:
Well spotted, but it's OK here. The idea is that importable modules are in this file. Apparently pre-commit is also used that way, so that's why it is here *as well*.